src/js/apps/patients/patient/dashboard/add-action_app.js

```diff
@@ -0,0 +1,65 @@
+import $ from 'jquery';
+import anime from 'animejs';
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import hbs from 'handlebars-inline-precompile';
+
+import Droplist from 'js/components/droplist';
+
+import App from 'js/base/app';
+
+
+export default App.extend({
+  onBeforeStart() {
+    anime({
+      targets: '.button', // fixme
+      color: ['#0582DA', '#0582DA'],
+      loop: true,
+      easing: 'easeInOutSine',
+      duration: 2000,
+      direction: 'alternate',
+    });
+  },
```

Comment:
I _think_ this is close to the desired animation, but I couldn't get it to work on the element, so I just threw a temp selector in